spec/features/user_asks_question_spec.rb

```diff
@@ -0,0 +1,59 @@
+require 'rails_helper'
+
+describe "ask a question", type: :feature do
+
+  it "displays the asked question" do
+    login_and_ask_question
```

Comment:
When you think about feature specs, think of them as a big thing the user is doing.

So rather than having a user `login_and_ask_question` five times consider doing it once and looking for the content you want to find one after another.